<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>videoshow_sel</class>
 <widget class="QWidget" name="videoshow_sel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="m_pBtn_Pause">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>700</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/Video/videoplayer_pause.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="m_pSlider_Progress">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>690</y>
     <width>630</width>
     <height>70</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_pLb_Time">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>690</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size: 32px;font: bold;</string>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pBtn_Return">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>690</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tu/Res/returnl.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
